src/sentry/incidents/endpoints/organization_alert_rule_details.py

```diff
@@ -130,12 +144,119 @@ def remove_alert_rule(request: Request, organization, alert_rule):
+A list of triggers, where each trigger is an object with the following fields:
+- `label`: One of `critical` or `warning`. A `critical` trigger is always required.
+- `alertThreshold`: The value that the subscription needs to reach to trigger the
+alert rule.
+- `actions`: A list of actions that take place when the threshold is met. Set as an empty list if no actions are to take place.
+```json
+triggers: [
+    {
+        "label": "critical",
+        "alertThreshold": 100,
+        "actions": [
+            {
+                "type": "email",
+                "targetType": "user",
+                "targetIdentifier": "23489853",
+                "inputChannelId": None
+                "integrationId": None,
+                "sentryAppId": None
+            }
+        ]
+    },
+    {
+        "label": "warning",
+        "alertThreshold": 75,
+        "actions": []
+    }
+]
+```
+Metric alert rule trigger actions follow the following structure:
+- `type`: The type of trigger action. Valid values are `email`, `slack`, `msteams`, `pagerduty`, `sentry_app`, `sentry_notification`, and `opsgenie`.
+- `targetType`: The type of target the notification will be sent to. Valid values are `specific`, `user`, `team`, and `sentry_app`.
+- `targetIdentifier`: The ID of the target. This must be an integer for PagerDuty and Sentry apps, and a string for all others. Examples of appropriate values include a Slack channel name (`#my-channel`), a user ID, a team ID, a Sentry app ID, etc.
+- `inputChannelId`: The ID of the Slack channel. This is only used for the Slack action, and can be used as an alternative to providing the `targetIdentifier`.
+- `integrationId`: The integration ID. This is required for every action type excluding `email` and `sentry_app.`
+- `sentryAppId`: The ID of the Sentry app. This is required when `type` is `sentry_app`.
+""",
+    )
+    environment = serializers.CharField(
+        required=False,
+        allow_null=True,
+        help_text="The name of the environment to filter by. Defaults to all environments.",
+    )
+    dataset = serializers.CharField(
+        required=False,
+        help_text="The name of the dataset that this query will be executed on. Valid values are `events`, `transactions`, `metrics`, `sessions`, and `generic-metrics`. Defaults to `events`. See [Metric Alert Rule Types under Create a Metric Alert Rule](/api/alerts/create-a-metric-alert-rule-for-an-organization/#metric-alert-rule-types) for valid configurations.",
+    )
+    queryType = serializers.ChoiceField(
+        required=False,
+        choices=((0, "event.type:error"), (1, "event.type:transaction"), (2, "None")),
+        help_text="The `SnubaQuery.Type` of the query. If no value is provided, `queryType` is set to the default for the specified `dataset.` See [Metric Alert Rule Types under Create a Metric Alert Rule](/api/alerts/create-a-metric-alert-rule-for-an-organization/#metric-alert-rule-types) for valid configurations.",
```

Comment:
I missed this in the "Create a Metric Alert..." review, but is "Snuba" an internal-only term? I don't think we want users to need to know what snuba is :)

This suggestion isn't great - would love to have a more descriptive first sentence, but I wasn't sure what that should be.

```suggestion
        help_text="The type of query. If no value is provided, `queryType` is set to the default for the specified `dataset.` See [Metric Alert Rule Types under Create a Metric Alert Rule](/api/alerts/create-a-metric-alert-rule-for-an-organization/#metric-alert-rule-types) for valid configurations.",
```

Mind changing this in the other endpoint as well?